src/main/java/org/broad/igv/ui/Main.java

```diff
@@ -91,25 +91,23 @@ public static void main(final String args[]) {
             setIgvDirectory(igvArgs);
         }
 
-        Runnable runnable = new Runnable() {
-            public void run() {
-
-                // This is a workaround for an internal JVM crash that was happening on Windows 10 (Creators Update).
-                // TODO: remove when enough users have migrated to Java 8u141 or greater.
-                // http://bugs.java.com/bugdatabase/view_bug.do?bug_id=JDK-8179014
-                if (Globals.IS_WINDOWS && System.getProperty("os.name").contains("10")) {
-                    UIManager.put("FileChooser.useSystemExtensionHiding", false);
-                }
-
-                DesktopIntegration.verifyJavaPlatform();
-                initApplication();
+        Runnable runnable = () -> {
 
-                JFrame frame = new JFrame();
-                frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
-                ImageIcon icon = new ImageIcon(Main.class.getResource("mainframeicon.png"));
-                if (icon != null) frame.setIconImage(icon.getImage());
-                open(frame, igvArgs);
+            // This is a workaround for an internal JVM crash that was happening on Windows 10 (Creators Update).
```

Comment:
This is a java 11 target, why the comment re java 8?

Reply:
We should perhaps ask @dbrant for this change (according to git blame, not our change). But happy to remove it, no worries.

Reply:
OK, why does this show as an addition vs master in the review tool?